src/components/shared/forms/styles/pillContainer.scss

```diff
@@ -0,0 +1,4 @@
+.pill-container{
+    display: flex;
```

Comment:
Looks like 4 space tabs here when it should have 2. Also the last `}` isn't aligned with the first.

Reply:
Changed.